lib/database/models/resto.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:objectbox/objectbox.dart';
+
+import 'promo.dart' show Promo;
+import 'user.dart' show User;
+
+@Entity()
+class Resto {
```

Comment:
I'd think in another name for this class. Maybe "Partners" or "vendor"?

Reply:
I like Partners. Nice